front/app/containers/Admin/projects/project/analysis/InputsList/index.tsx

```diff
@@ -24,15 +24,46 @@ const InputsList = () => {
     [data]
   );
 
-  const handleIntersection = useCallback(
-    (event: IntersectionObserverEntry, unobserve: () => void) => {
-      if (event.isIntersecting && hasNextPage) {
-        fetchNextPage();
-        unobserve();
-      }
-    },
-    [fetchNextPage, hasNextPage]
-  );
+  const inputsLength = inputs?.length || 0;
+  const parentRef = React.useRef<HTMLDivElement | null>(null);
+
+  const { getVirtualItems, getTotalSize, measureElement, scrollToIndex } =
```

Comment:
Just fyi, this is [the example](https://tanstack.com/virtual/v3/docs/examples/react/infinite-scroll) I used for inspiration.